pkg/diagnose/connectivity/trace_request.go

```diff
@@ -123,7 +120,7 @@ func verifyEndpointResponse(statusCode int, responseBody []byte, err error) {
 	}
 
 	statusString := color.GreenString("PASS")
-	if statusCode != http.StatusOK {
+	if statusCode >= http.StatusBadRequest {
```

Comment:
```suggestion
	if statusCode >= 400 {
```
if we're doing a comparison, then numbers are probably clearer (otherwise I have to remember that StatusBadRequest is 400 and then realize that this is looking for all 4xx or 5xx responses)